BuckarooSdk/Services/Bancontact/TransactionRequest/BanContactTransaction.cs

```diff
@@ -0,0 +1,47 @@
+﻿using BuckarooSdk.Transaction;
+
+namespace BuckarooSdk.Services.BanContact.TransactionRequest
+{
+    public class BanContactTransaction
+    {
+        /// <summary>
+		/// The configured transaction
+		/// </summary>
+		private ConfiguredTransaction ConfiguredTransaction { get; set; }
+
+        internal BanContactTransaction(ConfiguredTransaction baseTransaction)
+        {
+            this.ConfiguredTransaction = baseTransaction;
+        }
+
+        /// <summary>
+		/// The pay function creates a configured transaction with a BanContactPayRequest, 
+        /// that is ready to be executed.
+		/// </summary>
+		/// <param name="request">A BanContactPayRequestPayRequest</param>
+		/// <returns></returns>
+		public ConfiguredServiceTransaction Pay(BanContactPayRequest request)
+        {
+            var parameters = ServiceHelper.CreateServiceParameters(request);
+            var configuredServiceTransaction = new ConfiguredServiceTransaction(this.ConfiguredTransaction.BaseTransaction);
+            configuredServiceTransaction.BaseTransaction.AddService("Bancontactmrcash", parameters, "pay");
```

Comment:
Maybe it is nicer to create constant strings for the 2 string parameters ("Bancontactmrcash", "pay")?

Reply:
Also casing seems to be different on Buckaroo doc:

`"ServiceList": [
      {
        "Name": "bancontactmrcash",
        "Action": "Pay"
      }`

Reply:
I follow the pattern as for other implemented by Buckaroo transactions, eg: IdealProcessingTransaction.

Reply:
The string will be additional line(s) but i am not sure if it will improve readibility.